index.js

```diff
@@ -21,3 +20,94 @@ menuItem.addEventListener('click', () => {
+const projectCard = document.querySelectorAll('.project-card');
+const section = document.createElement('section');
+section.className = 'default';
+document.body.appendChild(section);
+const windowSize = window.matchMedia('(min-width: 992px)');
+
+function displayProjectCard(i) {
+  let projectTitle = projectCard[i].children[0];
+  const projectTechnologies = projectCard[1].children[3];
+
+  if (i === 0) {
+    projectTitle = document.querySelector('#project-title-1');
+  }
+
+  const project = [{
+    closePopup: './Icons/Enabled.png',
+    title: `${projectTitle.innerHTML}`,
+    technologies: `${projectTechnologies.innerHTML}`,
+    popupImage: './images/popup-image.png',
+    description: `<p>Lorem Ipsum is simply dummy text of the printing and typesetting industry. 
+                      Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, 
+                      when an unknown printer took a galley of type and scrambled it to make a 
+                      type specimen book. It has survived not only five centuries, but also the leap 
+                      into electronic typesetting, remaining essent</p>`,
+    popupHeader: `<div class="popup-header"> 
+                      <div class="title-style">${projectTitle.innerHTML}</div> 
+                      <img class="close-icon" src="./Icons/Enabled.png">
+                  </div>`,
+    popupTechnologies: ` <div id="projectTech" class="technologies">${projectTechnologies.innerHTML}</div>`,
+    popupDiscription: `<div class="popup-discription"> 
+                          <img class="popup-image" src= "./images/popup-image.png">
+                          <div class="popup-information">
+                              <div id="projectDiscription">
+                              <p>Lorem Ipsum is simply dummy text of the printing and typesetting industry. 
+                              Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, 
+                              when an unknown printer took a galley of type and scrambled it to make a 
+                              type specimen book. It has survived not only five centuries, but also the leap 
+                              into electronic typesetting, remaining essent</p> 
+                              </div>
+                              <div id="infoButton">
+                                  
+                                   <a href="https://biruk-bereka.github.io/Project-2-Portfolio/" target="_blank"><button class="button popupButton"> See Live<img class="icon-info" src="./Icons/Icon-Export.png"></button></a>
+                                  <a href="https://github.com/biruk-bereka/Project-2-Portfolio" target="_blank"><button class="button popupButton"> See Source<img class="icon-info" src="./Icons/Group.png"></button></a>
+                              </div>
+                          <div>
+                      </div>`,
+  }];
```

Comment:
- [ ] As required by the previous reviewer asked, you have to add more than one object in the array, you have 6 cards on your page so you have to add 6 objects in the array every one with different data and use them for displaying the data on the popup window.